spring-ai-2/src/test/java/com/baeldung/springai/evaluator/LLMResponseEvaluatorLiveTest.java

```diff
@@ -0,0 +1,113 @@
+package com.baeldung.springai.evaluator;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.ai.chat.client.ChatClient;
+import org.springframework.ai.chat.client.advisor.QuestionAnswerAdvisor;
+import org.springframework.ai.chat.model.ChatResponse;
+import org.springframework.ai.document.Document;
+import org.springframework.ai.evaluation.EvaluationRequest;
+import org.springframework.ai.evaluation.EvaluationResponse;
+import org.springframework.ai.evaluation.FactCheckingEvaluator;
+import org.springframework.ai.evaluation.RelevancyEvaluator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Import;
+
+import java.util.List;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+@SpringBootTest
+@Import(TestcontainersConfiguration.class)
+class LLMResponseEvaluatorLiveTest {
+
+    @Autowired
+    private ChatClient contentGenerator;
+
+    @Autowired
+    private RelevancyEvaluator relevancyEvaluator;
+
+    @Autowired
+    private FactCheckingEvaluator factCheckingEvaluator;
+
+    @Test
+    void whenChatClientProvidesAnswerRelevantToTopic_thenRelevancyEvaluationSucceeds() {
+        String question = "How many sick leaves can I take?";
```

Comment:
Rephrase so it reads more naturally:
```suggestion
        String question = "How many days sick leave can I take?";
```

Reply:
Feedback incorporated.